eslint.config.js

```diff
@@ -355,5 +349,16 @@ export default [
 			"max-len": "off",
 			"one-var": "off"
 		}
+	},
+
+	{
+		files: [
+			"build/**/*.js",
+			"dist/jquery.bundler-require-wrapper.js",
+			"dist/jquery.bundler-require-wrapper.slim.js"
+		],
+		languageOptions: {
+			sourceType: "commonjs"
```

Comment:
Does this mean ESLint is not following the `type` field from `package.json`?

Reply:
Yea, eslint doesn't seem to respect the extra package.json files